chemistry_lab.install

```diff
@@ -0,0 +1,666 @@
+                'not null' => TRUE
+            ),
+          'pincode' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '10',
+                'not null' => TRUE
+            ),
+          'state' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '100',
+                'default value' => 'NONE',
+                'not null' => TRUE
+            ), 
+          'country' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '100',
+                'default value' => 'NONE',
+                'not null' => TRUE
+            ),  
+          'options' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '200',
+                'not null' => TRUE
+            ), 
+          'project_guide_name' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '255',
+                'not null' => TRUE
+            ),
+          'project_guide_department' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'text',
+                'not null' => TRUE
+            ),
+          'project_guide_email_id' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '255',
+                'default value' => 'NULL',
+                'not null' => TRUE
+            ),
+          'project_title' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '255',
+                'not null' => TRUE
+            ),
+          'description' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'text',
+                'not null' => TRUE
+            ),
+          'software_version' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '50',
+                'not null' => TRUE
+            ),
+          'directory_name' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'varchar',
+                'length' => '255',
+                'not null' => TRUE
+            ),
+          'approval_status' => array(
+                'description' => t('TODO: please describe this field!'),
+                'type' => 'int',
+                'length' => '11',
+                'not null' => TRUE
+            ),
+          'is_submitted' => array(
```

Comment:
Please check if this field is used in the code, if not remove the field from table